src/js/component.js

```diff
@@ -152,7 +151,6 @@ class Component {
         this.el_.parentNode.removeChild(this.el_);
       }
 
-      DomData.removeData(this.el_);
```

Comment:
We don't have to do this, once `this.el_` becomes null the weakmap will remove it on its own.

Reply:
I don't think we should be relying on the behavior of the garbage collector, when we're done using it we should empty it out.